esp/templates/survey/answers.html

```diff
@@ -59,6 +62,13 @@
                     <li>Average for all classes: {{ q.question.global_average }}</li>
                 {% endif %}
             </ul>
+            <!-- Creates a key that provides details about the Histogram Labeling-->
+            <b>Response Labels:</b>
```

Comment:
Looks like now this shows up for numeric rating questions, but we don't have any labels to loop through. Maybe it would make sense to move the lower_text/middle_text/upper_text to here (and remove it from lines 39-46 above). You'll need an if/else here to check if the question has each of the texts (or a list instead if it's a labeled numeric rating question).